### DISCUSSION TITLE
Why Anime Watch Progress Isn't Saved Locally for Logged-Out Users

### DISCUSSION THREAD

**Discussion by lovebrownie** [aleganza/akuse#90] [General]
I was watching an anime but noticed that it wasn't saved locally because I wasn't logged in.

**Comment by aleganza** [comment]
episodes minute watch progress is not available
if you log in, you can keep track only the episodes you've watched, but you can't resume from where you left an episode.
:)

**Reply by lovebrownie** [reply]
Why you don't save episodes watch progress?

**Reply by aleganza** [reply]
I didn't add it

**Reply by DomDory** [reply]
Do you plan on adding it?

**Reply by aleganza** [reply]
With the v1.6.0 it's been added!

**Reply by DomDory** [reply]
Omg you are amazing thank you

**Reply by aleganza** [reply]
@ch0nker added it!
